azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/VirtualMachine.java

```diff
@@ -482,13 +482,56 @@
     }
 
     /**
+     * The virtual machine definition stage with availability set.
+     *
+     * @param <T> The virtual machine definition in creatable stage.
+     */
+    interface DefinitionWithAvailabilitySet<T extends DefinitionCreatable> {
+        /**
+         * Specifies the name of the availability set to create, the virtual machine will be part of
+         * this availability set.
```

Comment:
according to the convention I've seen elsewhere and trying to follow myself - only the top description starts with capital letters (and ends with periods). The param and return descriptions should start with a small letter (and no period). For example, see how File is javadoc'd.